airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -118,46 +121,56 @@ export const DagsList = ({ cardView = false }) => {
     [pagination, setTableURLState],
   );
 
+  const handleSearchBarChange = useCallback<ChangeEventHandler<HTMLInputElement>>(
+    ({ currentTarget: { value } }) => {
+      searchParams.set(NAME_PATTERN_PARAM, value);
+      setSearchParams(searchParams);
+      setTableURLState({
+        pagination: { ...pagination, pageIndex: 0 },
+        sorting,
+      });
+    },
+    [dagDisplayNamePattern],
+  );
+
   return (
     <>
-      {isLoading ? <Spinner /> : undefined}
```

Comment:
You will probably need to rebase soon since we moved where loading is handled over here: https://github.com/apache/airflow/pull/42711

Reply:
I will wait for the other PR is merge i will rebased onto this one :)

Reply:
It's merged. Feel free to rebase now.

Reply:
I pushed again it should be ready to be reviewed again.

Reply:
i did a force push instead of a rebase at the time thinking it would be easier. If this is not standard practice lmk i can created another PR with a rebase instead.